autoload/test/go/ginkgo.vim

```diff
@@ -0,0 +1,50 @@
+if !exists('g:test#go#ginkgo#file_pattern')
+  let g:test#go#ginkgo#file_pattern = '\v[^_].*test\.go$'
+endif
+let test#go#ginkgo#patterns = {
+  \ 'test': ['\v^\s*It\("(.*)",', '\v^\s*Context\("(.*)",', '\v.*Describe\("(.*)",'],
+  \ 'namespace': [],
+\}
+
+function! test#go#ginkgo#test_file(file) abort
+  if fnamemodify(a:file, ':t') =~# g:test#go#ginkgo#file_pattern
+    if exists('g:test#go#runner')
```

Comment:
`go test` and ginkgo test files share the same filename pattern. Even if we `search("github.com/onsi/ginkgo")` here, it is still necessary to use `test#go#runner` to force using one of the runners.

I think we should keep using `test#go#runner` and replace `executable("ginkto")` with `search("github.com/onsi/ginkgo")`. Is this OK?